docs/forms.rst

```diff
@@ -137,6 +137,14 @@ method with the convention `validate_fieldname`::
             if field.data < 13:
                 raise ValidationError("We're sorry, you must be 13 or older to register")
 
+The same principle applies for filters with the convention `filter_fieldname`::
```

Comment:
I know filters are documented elsewhere, but this should probably contain a sentence like "Filters are applied after processing the default and incoming data, but before validation.".